### DISCUSSION TITLE
Will .albumart, .covers and .waveforms be auto-cleaned when media is removed/changed?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6099] [Q&A] [answered]
By chance (using SFTP), I found that my `.albumart` folder is already 10.2 GB, my `.covers` another 2.1 GB in size.

From bad experiences with other software, my question is now: _Will these be automatically cleaned when removing media?_  Or when re-scanning and some cover images in the media have changed?

Wouldn’t really like an ever-growing bin of trash…

**Comment by BusterNeece** [comment, marked as answer]
They do get routinely cleaned up. See `App\Sync\Task\CleanupStorageTask` for more details.

**Comment by Moonbase59** [comment]
Perfect, thanks for the conformation!
